src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -3,28 +3,49 @@
 --********************************************************************/
 
 using System;
+using System.Collections;
+using System.Collections.Generic;
 using System.Collections.ObjectModel;
+using System.Globalization;
+using System.IO;
+using System.Linq;
 using System.Management.Automation;
 using System.Net;
-using System.IO;
-using System.Text;
-using System.Collections;
-using System.Globalization;
+using System.Net.Http;
+using System.Net.Http.Headers;
 using System.Security;
 using System.Security.Authentication;
 using System.Security.Cryptography;
 using System.Security.Cryptography.X509Certificates;
-using Microsoft.Win32;
-using System.Net.Http;
-using System.Net.Http.Headers;
+using System.Text;
+using System.Text.RegularExpressions;
 using System.Threading;
 using System.Xml;
-using System.Collections.Generic;
-using System.Text.RegularExpressions;
-using System.Linq;
+using Microsoft.Win32;
```

Comment:
Do we really need this?

Reply:
It looks like we don't.  Should I remove it in this PR or in another?

Reply:
Better in next - the PR is very large :-)

Reply:
I have added it to the task list in #5610